src/Apps/Order/Routes/Shipping2/Hooks/useShippingContext.ts

```diff
@@ -1,12 +1,13 @@
-import { useContext, useState } from "react"
-import { FormikProps } from "formik"
+import { useContext, useReducer } from "react"
 import { ShippingProps } from "Apps/Order/Routes/Shipping2"
 import { useParseOrderData } from "Apps/Order/Routes/Shipping2/Hooks/useParseOrderData"
 import {
   ShippingContext,
   ShippingContextProps,
+  ShippingStage,
+  State,
 } from "Apps/Order/Routes/Shipping2/Utils/ShippingContext"
-import { FulfillmentValues } from "Apps/Order/Routes/Shipping2/Utils/shippingUtils"
+import { useShippingContextHelpers } from "Apps/Order/Routes/Shipping2/Hooks/useShippingContextHelpers"
```

Comment:
The `ShippingContext` is meant to provide all of the shared means to support itself, and is already global to the route. The logic in this file should be returned from the context itself, so that any helper could be retrieved via 

```tsx
const { doThing } = useShippingContext() 
```

Reply:
If there are some utils (we don't use the term "helpers" in force, that's a rails idiom), its fine to group them into their own file (under "/utils") and import them into the context in order to funnel them to the rest of the app via the context, but I don't think that's necessary here. We can just keep everything co-located and refactor it all later.

Reply:
Similar to my below comment, i'm open to changing this later but am going to merge this once tests pass.